deploy/Chart/templates/controller/deployment.yaml

```diff
@@ -39,6 +39,10 @@ spec:
           value: '/var/tls/cert'
         - name: PORT
           value: '9443'
+        {{- if .Values.global.caCert.filename }}
+        - name: SSL_CERT_DIRS
```

Comment:
Can you leave a comment here that says this specific env-var is read by Go?

The reason why I'm asking is that line 38 defines `TLS_CERT_DIR`. Someone might come along in the future and *make it consistent* which would break this functionality.

Reply:
added comments